c10/util/sparse_bitset.h

```diff
@@ -442,7 +442,8 @@ class SparseBitVector {
 
   SparseBitVector(const SparseBitVector& RHS)
       : Elements(RHS.Elements), CurrElementIter(Elements.begin()) {}
-  SparseBitVector(SparseBitVector&& RHS)
+  SparseBitVector(SparseBitVector&& RHS) noexcept(
```

Comment:
This is always true except for MSVC. The scenario where std::list can throw on assignment is very rare though (OOM) so maybe we can just hardcode it as noexcept. This change is strictly better than what was there before though.